chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -27,15 +27,17 @@ spec:
           - "--config.file=/alertmanager.yml"
           - "--storage.path=/alertmanager"
         livenessProbe:
-          httpGet:
-            path: /#/status
-            port: 9093
-          timeoutSeconds: 30
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - wget --quiet --tries=1 --spider http://localhost:9093/-/ready || exit 1
```

Comment:
So we had a timeout of 30 seconds. What is the implicit timeout of wget and can we also configure that to the same value here?

Reply:
```
	-T SEC		Network read timeout is SEC seconds
```